app/src/main/java/com/infomaniak/mail/ui/main/folder/ThreadListFragment.kt

```diff
@@ -148,6 +148,11 @@ class ThreadListFragment : Fragment(), SwipeRefreshLayout.OnRefreshListener {
         updateSwipeActionsAccordingToSettings()
     }
 
+    override fun onPause() {
+        mainViewModel.isDownloadingChanges.value = false to null
+        super.onPause()
+    }
```

Comment:
Seems to be unnecessary

Reply:
Not unnecessary, but not the right fix too.